### DISCUSSION TITLE
Error message when trying to arm alarm after recent upgrade of Home Assistant

### DISCUSSION THREAD

**Discussion by ironicent** [tsightler/ring-mqtt#847] [Support Q&A]
After performing an upgrade to the most recent version of Home Assistant. I am getting the attached error message when trying to arm the alarm.  I can arm the alarm without entering a code after providing a disarm code in the configuration of the addon, but I am then required to enter the disarm code to disarm the alarm. Before the upgrade, I was able to arm and disarm the alarm without entering a code in the addon configuration.

Current versions:
Core 2024.6.0
Supervisor 2024.06.0
Operating System 12.3
Frontend 20240605.0
Ring-MQTT 5.6.4

![Screenshot 2024-06-06 at 2 51 36 PM](https://github.com/tsightler/ring-mqtt/assets/6020312/f59d1ead-e20e-462d-b3c8-d969f7e25e5e)

**Comment by tsightler** [comment]
Please search for existing issues/discussions prior to posting, this is a dupe of:

https://github.com/tsightler/ring-mqtt/issues/846
https://github.com/tsightler/ring-mqtt/discussions/843
